src/styles/blocks/stars.css

```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  margin-right: 4px;
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(-n + 5) {
+  background-image: url(../../images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
`background-position` and `background-repeat` properties are already existed in the initial star class, you shouldn't repeat them here